src/modes/single/SingleLocation.svelte

```diff
@@ -11,20 +11,28 @@
   } from '../../stores';
   import { sensorList } from '../../stores/constants';
   import SensorCard from './SensorCard.svelte';
-  import throttle from 'lodash-es/throttle';
+  import debounce from 'lodash-es/throttle';
```

Comment:
I just notice that: you call it debounce but import throttle? was this intentional?

Reply:
It was unintentional in the sense that I assumed debounce was part of throttle, and mysteriously, it worked.  However, I see there is a separate debounce.js, so changing the dependency to that.  This resulted in vscode complaining "Could not find declaration file for module 'lodash-es/debounce'" and the suggestion to do npm install @types/lodash-es, which I did, and it changed a couple of json config files.  Those changes are now pushed.  Let me know if this is not right.